travel_temp.html

```diff
@@ -128,4 +131,4 @@ <h2>Trip to New Delhi</h2>
         <script src="assets/js/travel_temp.js"></script>
     </body>
 
-</html>
\ No newline at end of file
+</html>
```

Comment:
Do not make these changes as it would cause merge conflict for others!

Reply:
That had a bad character ending :') but I'll revert that surely